src/iop/temperature.c

```diff
@@ -938,10 +938,16 @@ void reload_defaults(dt_iop_module_t *module)
   // we might be called from presets update infrastructure => there is no image
   if(!module->dev) goto end;
 
+  int is_raw = dt_image_is_raw(&module->dev->image_storage);
+
+  // White balance module doesn't need to be anbled for monochrome raws (like
+  // for leica monochrom cameras)
+  if(is_raw && is_leica_monochrom(&(module->dev->image_storage))) goto gui;
+
   if(module->gui_data) prepare_matrices(module);
```

Comment:
So what happens if this is not called?

Reply:
Did you mean if `reload_defaults` isn't called, or if `prepare_matrices` isn't called?

Monochrom raws don't have colour matrices anyway,  so all `prepare_matrices` does for monochrom raws is write an error messas.

I'll fixup the other stuff.

Reply:
If prepare_matrices isn't called.

> Monochrom raws don't have colour matrices anyway, so all prepare_matrices does for monochrom raws is write an error messas.

Ah, now that i checked, the temperature/tint slider is already non-functional.